source/concepts/xml-files.rst

```diff
@@ -0,0 +1,57 @@
+XML Files
+==============================================
+
+The XML format for serializing OpenMM simulations consists of a root element ``<OpenMMSimulation>`` containing three main components:
+
+1. **Starting Structure**: Enclosed in either ``<pdbx>`` or ``<pdb>`` tags
+2. **OpenMM Serialized System**: Enclosed in ``<System>`` tags
+3. **OpenMM Serialized Integrator**: Enclosed in ``<Integrator>`` tags
+
+An optional fourth component, the OpenMM serialized state, may also be included.
```

Comment:
Can you describe more about what this is? What type of object is it and what is the tag?

Reply:
Done in this commit [84ca9a9](https://github.com/IRL2/nanover-docs/pull/115/commits/84ca9a93f9ad2f5f4ace78b0b3f69b84b985e283)